tests/functional_tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Making `tests/functional_tests` a package seems unneccessary.

Reply:
The frontend tests and the iterator tests use the same test data (https://github.com/GridTools/gt4py/blob/cb84312eb0c6ef557405856be3f27592b7247818/tests/functional_tests/ffront_tests/test_math_builtin_execution.py#L20). After discussion with @egparedes I opted for relative imports (consequently requiring the init files). Using conftest is also a possibility, but placing the data in `tests/conftest.py` is not appealing.